### ISSUE TITLE
tfcompile of a fitted keras model

### ISSUE THREAD

**Issue by volvador** [tensorflow/tensorflow#22903]
Hello,
I want to define a simple model in keras, fit it, then tfcompile it.

    import tensorflow as tf
    from tensorflow.python.keras.models import Sequential
    from tensorflow.python.keras.layers import Dense
    model = Sequential()
    model.add(Dense(input_dim=X.shape[1], units=hidden_dim))
    model.add(Dense(units=Y.shape[1], activation='linear'))
    model.compile(loss='mse', optimizer='adam')
    model.fit(X, Y, batch_size=100)

Can you please give an example of how to tfcopile this fitted model, or at least give some steps (for example, should the graph be frozen before using tfcompile?)

**Comment by volvador** [comment]
OK, I figured out that I needed to freeze the Keras graph before calling the tfcompile
So I do this:

    session = tf.keras.backend.get_session()
    output_node_names = [node.op.name for node in model.outputs]
    graphdef = tf.graph_util.convert_variables_to_constants(session, session.graph_def, output_node_names)
  
 I need indeed to transform variables to constants> Now how can I get a graph from the graphdef, and how can I get the input and output of that graph. Here is what I did, but does not seem to work

    with tf.Graph().as_default() as graph:
        tf.import_graph_def(graphdef)
        inp = graph.get_tensor_by_name(graph.get_operations()[0].name+':0')
        out = graph.get_tensor_by_name(graph.get_operations()[-1].name+':0')

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

If you think we've misinterpreted a bug, please comment again with a clear explanation, as well as all of the information requested in the [issue template](https://github.com/tensorflow/tensorflow/issues/new). Thanks!
